### PR TITLE
Remove affectedDumpFile if dumping is enabled

### PR DESCRIPTION
**Pull request by pdalpra**

### CODE REVIEW

File: docker/crane-env.go [hunk 1]
```diff
@@ -48,6 +48,11 @@ func (rs ignoreCommentSorter) Less(i, j int) bool {
 func (c *RichClient) CraneEnvForBranch(r Registry, branch string, useShaTags bool, dumpAffected bool, affectedDumpFile string) ([]string, error) {
 	var merr = multierror.New()
 	var resultEnvVars []string
+	if dumpAffected {
+		if err := os.Remove(affectedDumpFile); err != nil {
+			return nil, fmt.Errorf("Failed to removed %s: %v", affectedDumpFile, err)
```
Comment (PhilippeDupont) [inline docker/crane-env.go]: Don't you want to only remove the file if it exists and continue if not ?

Comment (pdalpra) [inline docker/crane-env.go]: So that's why that test is failing :D

Comment (pdalpra) [inline docker/crane-env.go]: I can check that the error is not caused by the file not already existing, LGTY ?

Comment (PhilippeDupont) [inline docker/crane-env.go]: I prefer this yes ;)

**Comment by pdalpra** [comment]
@PhilippeDupont all good now :)

**Comment by PhilippeDupont** [comment]
LGTM

**Pull request merged**